samples/net/openthread/coap_server/src/ot_coap_utils.c

```diff
@@ -0,0 +1,247 @@
+#include <logging/log.h>
+LOG_MODULE_REGISTER(ot_coap_utils, CONFIG_OT_COAP_UTILS_LOG_LEVEL);
+
+#include "ot_coap_utils.h"
+
+#include <net/net_pkt.h>
+#include <net/net_l2.h>
+#include <net/openthread.h>
+#include <openthread/message.h>
+#include <openthread/ip6.h>
+#include <openthread/coap.h>
+#include <openthread/thread.h>
+
+typedef struct {
+	struct otInstance *ot;
+	bool provisioning_enabled;
+	light_request_callback on_light_request;
+	provisioning_request_callback on_provisioning_request;
+} server_context_t;
+
+static server_context_t server_context = { .ot = NULL,
+					   .provisioning_enabled = false,
+					   .on_light_request = NULL,
```

Comment:
please double check it, but I think there was a warning about null initialized static variables.

Reply:
I don't see any warnings